### PR TITLE
Feature/fix msbuild props injection

### LINKED ISSUE #4471: Regression in MSBuild helper properties injection
Unfortunately, in #4113 we have missed the fact the `ForceImportBeforeCppTargets` parameter requires an absolute path to the properties files to be injected.

### PR DESCRIPTION
**Pull request by Minimonium**
Fixes #4471
Also removes one redundant test.

I have a PEP8 embedded into VSCode. Should I also use it to auto-format files affected by PRs on the way?

### CODE REVIEW

File: conans/test/unittests/client/build/msbuild_test.py [hunk 1]
```diff
@@ -246,3 +237,19 @@ def verbosity_explicit_test(self):
...
 
+    def properties_injection_test(self):
+        settings = MockSettings({"build_type": "Debug",
+                                 "compiler": "Visual Studio",
+                                 "arch": "x86_64"})
+        conanfile = MockConanfile(settings)
+        msbuild = MSBuild(conanfile)
+        command = msbuild.get_command("dummy.sln", props_file_path="conan_build.props")
+
+        print(command)
```
Comment (memsharded) [inline conans/test/unittests/client/build/msbuild_test.py]: Please don't use prints in tests.

File: conans/test/unittests/client/build/msbuild_test.py [hunk 2]
```diff
@@ -131,17 +133,6 @@ def get_version_test(self):
         self.assertRegexpMatches(version, "(\d+\.){2,3}\d+")
         self.assertGreater(version, "15.1")
 
-    def custom_properties_test(self):
```
Comment (lasote) [inline conans/test/unittests/client/build/msbuild_test.py]: Why removing this test?

Comment (Minimonium) [reply conans/test/unittests/client/build/msbuild_test.py]: It's defined twice. With the same name and the same body.

File: conans/test/unittests/client/build/msbuild_test.py (hunk 1 above)
Comment (Minimonium) [reply conans/test/unittests/client/build/msbuild_test.py]: My bad.

Comment (Minimonium) [reply conans/test/unittests/client/build/msbuild_test.py]:
```suggestion
```

File: conans/test/unittests/client/build/msbuild_test.py (hunk 2 above)
Comment (Minimonium) [reply conans/test/unittests/client/build/msbuild_test.py]: See https://github.com/conan-io/conan/blob/b70712772e6d6615750b615f4cb3d4b472b9c273/conans/test/unittests/client/build/msbuild_test.py#L48

**Comment by danimtb** [comment]
Seems the github credentials are not working in CI. We will fix and relaunch the CI shortly. Thanks for the PR!

**Comment by memsharded** [comment]
Thanks very much @Minimonium for contributing this PR. It has been superseded by https://github.com/conan-io/conan/pull/4488, which is basically what you did, but targeted to the patch release 1.12.2.

**Pull request closed without merging**